admin/filters/docLink.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+function before_filter_docLink($data, $folder) {
+    return preg_replace_callback('/:doc:`(.*?) <(.*?)>`/', function ($matches) {
+        $text = $matches[1];
+        $url  = $matches[2];
+
+        // Check if the URL doesn't contain '..'
+        if (! str_contains($url, '..')) {
+            $url = explode('/', $url);
+            $url = array_pop($url);
```

Comment:
Why `array_pop()`? 
This does not seem to work.

```
docs-rst//helpers/cookie_helper.rst:    cookies. Refer to the :doc:`Response Library </outgoing/response>` for
```

Reply:
I was testing this based on URLs from `models/model`. From what I tested :doc: have to be translated this way:
```
:doc:`Entity class </models/entities>` -> `Entity class <model.md>`_
:doc:`Query Builder <../database/query_builder>` -> `Query Builder <../database/query_builder.md>`_
```

But there may be other scenarios, so I will look into this. Thank you for pointing that out.

Reply:
There seems to be three patterns of paths to files in RST files:
- `/foo/bar`
- `../foo/bar`
- `./foo/bar`

If there are problems, there is a way to normalize the path in the RST files first.